content/rc/administration/setup/create-subscription.md

```diff
@@ -14,9 +14,11 @@ When you create a Redis Cloud subscription, you must choose:
 - A cloud provider: **Amazon AWS**, **Microsoft Azure**, **Google Cloud Platform**
 - The provider region
 - Redis Cloud service level:
-    - **Essentials** - For development environments and low-throughput applications
-    - **Pro** - For high-throughput applications, many databases or large datasets
-    - **Ultimate** - For a more complete support experience (Contact [Redis Labs Sales](https://redislabs.com/redis-enterprise-cloud/pricing/))
+    - **Fixed** - Set pricing designed for low-throughput applications  
+    - **Flexible**  - "Pay as you go" plans for any dataset size or throughput  
+    - **Annual** - Predefined annual consumption commitments that provide substantial discounts over Flexible plans  
```

Comment:
Do we highlight "Flexible" in any way?

Reply:
I don't think so here, as it doesn't refer to UX.